jerry-core/ecma/operations/ecma-proxy-object.c

```diff
@@ -404,56 +404,60 @@ ecma_proxy_object_set_prototype_of (ecma_object_t *obj_p, /**< proxy object */
     return ecma_op_ordinary_object_set_prototype_of (target_obj_p, proto);
   }
 
+  /* 8 */
   ecma_object_t *func_obj_p = ecma_get_object_from_value (trap);
   ecma_value_t args[] = { target, proto };
 
-  /* 9. */
   ecma_value_t trap_result = ecma_op_function_call (func_obj_p, handler, args, 2);
 
   ecma_deref_object (func_obj_p);
 
-  /* 10. */
   if (ECMA_IS_VALUE_ERROR (trap_result))
   {
     return trap_result;
   }
 
+  /* 9. */
   bool boolean_trap_result = ecma_op_to_boolean (trap_result);
 
   ecma_free_value (trap_result);
 
+  /* 9. */
```

Comment:
The step "9" is mentioned twice. Do we need it twice?

Reply:
Yes, since the 9th step is different in the ES6 and ES11 standard.